src/mmg2d/analys_2d.c

```diff
@@ -524,8 +524,8 @@ int MMG2D_regnor(MMG5_pMesh mesh) {
   maxit = 10;
   res0 = 0.0;
   nn = 0;
-  lm1 = 0.4;
-  lm2 = 0.399;
+  lm1 = mesh->info.lxreg;
+  lm2 = 0.99*lm1;
```

Comment:
Is there a specific reason to choose to shift slightly `lm2` from its theoretical value using a multiplication instead of substracting a small value (1e-3)? It means that the shifting value will be greater for high values of `lm1` (`1e-2` for `lm1=1`) and that we will have no shift for `lm1=0` : is this wanted?
Thanks by advance.